backend/migrations/01-remove-project-unique-index.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose')
+const Promise = require('bluebird')
+const logger = require('../misc/logger')
+
+module.exports = {
+    index: 1,
+    name: '01-remove-project-unique-index',
+    description: 'Remove unique team and event index from Project schema',
```

Comment:
Hmm, what's the purpose of this?

Reply:
It removes an index that was defined on Projects before. The index made sure that there would be only one Project per team and per event. Since we now allow multiple projects per team and event, this index needs to be removed.